addons/point_of_sale/views/pos_report_printer_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="pos_report_printer_view_tree" model="ir.ui.view">
+        <field name="name">pos.report_printer.view.tree</field>
+        <field name="model">pos.report_printer</field>
+        <field name="arch" type="xml">
+            <tree string="Report Printers" editable="top">
```

Comment:
Let's change it to `"Report Email Printing"` to make a clear distinction between the printing through IoT and this new way of printing